src/core/modules/MinorPlanet.hpp

```diff
@@ -90,17 +88,20 @@ class MinorPlanet : public Planet
 	//! @param slope Slope parameter G. This is usually [0..1], sometimes slightly outside. Allowed here [-1..2].
 	void setAbsoluteMagnitudeAndSlope(const float magnitude, const float slope);
 
-	//! renders the subscript in a minor planet provisional designation with HTML.
-	//! \returns an empty string if the source string is not a provisional
-	//! designation.
-	static QString renderProvisionalDesignationinHtml(QString plainText);
+	//! renders the subscript in a minor planet IAU provisional designation with HTML.
+	static QString renderIAUDesignationinHtml(QString plainText);
```

Comment:
please use string references (QString &plainText) [everywhere]

Reply:
Why?

Reply:
I suppose Georg meant using `const` references, rather than what he wrote. A small advantage of this is avoidance of the atomic check-and-increment of the refcount inside the Qt's implicit sharing mechanism (I doubt this would make any measurable difference in this case, but it makes sense to use const reference by default in such situations).

Reply:
Indeed, `(const QString &arg)` is even clearer. See e.g. https://www.ibm.com/docs/en/zos/2.4.0?topic=calls-pass-by-reference-c-only
> Pass-by-references is more efficient than pass-by-value, because it does not copy the arguments.

This may not mean a lot for simple types (int, double, ...) but seems recommended for QStrings and other complex types. I try to update these things as I find them.